### PR TITLE
Add custombox w/ git auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10669

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/dixso/custombox/
 * Official website (optional, not the repository): https://dixso.github.io/custombox/
 * NPM package url (optional): https://www.npmjs.com/package/custombox
 * License and its reference: [MIT](https://github.com/dixso/custombox/#custombox----)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 39
   - Count of stars: 501
   - Count of forks: 144
 * NPM download stats (optional):
   - Downloads in the last day: 2
   - Downloads in the last week: 125
   - Downloads in the last month: 312

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by extend1994** [comment]
@pvnr0082t Here is the confirmation: dixso/custombox#225. According to this, I think we need ` custombox.legacy.min.js`. As for problem of the minified file, can I rename the file (`custombox.legacy.min.js` -> `custombox.legacy.js`) and minify it? But Here I think of another problem: even I do this, auto-update seems to minify files by recognizing the format of the file. So if the format is `.min.js` the file still won't be minified?

**Comment by pvnr0082t** [comment]
>auto-update seems to minify files by recognizing the format of the file

@extend1994 You're right, so you should ask the author if it's possible to change the naming scheme.

**Comment by extend1994** [comment]
@pvnr0082t I asked the author about this, once the author reply, I will report here.

**Comment by extend1994** [comment]
@pvnr0082t The author provided his minified `custombox.legacy.min.js` and I replace the old one with new official one, please help me review it, thanks!

File: ajax/libs/custombox/4.0.0/custombox.legacy.min.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/*
+ *  custombox - Modal dialog effects with transitions CSS3
+ *  version: 4.0.1
```
Comment (PeterDaveHello) [inline ajax/libs/custombox/4.0.0/custombox.legacy.min.js]: You commit a v4.0.1 file here?

**Comment by extend1994** [comment]
@PeterDaveHello Yes the version author released is v4.0.1. So I modified the commit, please help me review it, thank you!

**Pull request merged**